rootfs/templates/wrapper

```diff
@@ -187,7 +194,7 @@ function use() {
 		echo "# Starting new ${DOCKER_NAME} session from ${DOCKER_IMAGE}"
 		echo "# Exposing port ${GEODESIC_PORT}"
 		[ -z "${GEODESIC_DOCKER_EXTRA_ARGS}" ] || echo "# Launching with extra Docker args: ${GEODESIC_DOCKER_EXTRA_ARGS}"
-		docker run "${DOCKER_ARGS[@]}" ${GEODESIC_DOCKER_EXTRA_ARGS} ${DOCKER_IMAGE} -l $*
+		docker run "${DOCKER_ARGS[@]}" ${GEODESIC_DOCKER_EXTRA_ARGS} ${GEODESIC_ADDITIONAL_DOCKER_ARGS} ${DOCKER_IMAGE} -l $*
```

Comment:
In terms of naming, it's hard to distinguish `GEODESIC_DOCKER_EXTRA_ARGS` and `GEODESIC_ADDITIONAL_DOCKER_ARGS`, they mean the same thing